my_list.rb

```diff
@@ -0,0 +1,16 @@
+require_relative 'my_enumerable'
+
+class MyList
+  include MyEnumerable
+  def initialize(*numbers)
+    @list = numbers
```

Comment:
- [ ] I think the initialize method should look like this:
```
def initialize(*args)
        @list = []
        args.each { |arg| @list << arg }
    end
```
For now when we run the code, we have this error:
![image](https://user-images.githubusercontent.com/71874425/173129553-ffc7657c-1957-4c58-9aa8-29d50aeb8d8d.png)
Kindly, make sure that you initialize your MyList Class.

Reply:
- [ ] Hello @kemigabocatherine , I did an error while running your project for the first time, It seems that it was an error from my local PC. Everything works as expected, Kindly, submit your PR for re-review, and then the next reviewer could approve your PR.
![image](https://user-images.githubusercontent.com/71874425/173138024-7159299e-1bcb-403b-8b10-d5682a34cfb7.png)